SETUP.md

```diff
@@ -77,8 +77,17 @@ It is best to copy and paste it.
 ### Correct username and password does not work
 Continuum depends on
 [Android System Webview](https://play.google.com/store/apps/details?id=com.google.android.webview)
-by default for logging into Reddit. So if having the login issue, your best
-course of action would be to upgraded to the latest version of Android
+by default for logging into Reddit. So if having the login issue, you can as a workaround use the login button located at the bottom of your screen which opens up reddit in your default browser for you to login. Keep in mind this method tends to not work with hardened browsers like Fennec. Here are some of the browsers which are guaranteed to work with this method; 
```

Comment:
> So if having the login issue, you can as a workaround use the login button located at the bottom of your screen which opens up reddit in your default browser for you to login.

The wording could be better.

Suggestion:
One workaround for a login issue is to use the login button located at the bottom of your screen. It opens up reddit in your default browser for you to login.